includes/class-wc-install.php

```diff
@@ -146,6 +146,7 @@ class WC_Install {
 			'wc_update_product_lookup_tables',
 			'wc_update_400_increase_size_of_column',
 			'wc_update_400_db_version',
+			'wc_reset_action_scheduler_migration_status',
```

Comment:
Just noting here for future reference, that db update routines should be prefixed `wc_update_400_` and the routine that updates the db version (`wc_update_400_db_version`) should be the last one. Since we noticed this too late on the 4.0 release cycle, we opted to keep it as is and simply add a code comment documenting it.